doc/releases/changelog-0.11.0.md

```diff
@@ -177,21 +177,36 @@
       return circuit(x)
   ```
 
-* Changes to reduce compile time:
+  ```pycon
+  >>> func(0.1)
+  Array(0.99500417, dtype=float64)  
+  ```
+
+<h3>Improvements 🛠</h3>
 
-  - Turn off MLIR's verifier.
+* Pattern rewriting in the `quantum-to-ion` lowering pass has been changed to use MLIR's dialect conversion
+  infrastructure.
+  [(#1442)](https://github.com/PennyLaneAI/catalyst/pull/1442)
+
+* The `merge-rotations` peephole optimization pass has been extended to also merge compatible rotation 
+  gates (either both controlled, or both uncontrolled) where rotation angles are any combination of 
+  static constants or dynamic values.
+  [(#1489)](https://github.com/PennyLaneAI/catalyst/pull/1489)
```

Comment:
Sounds almost like a bug fix introduce by the splitting of the gates into static and dynamic (another PR).

Reply:
Just had another look, this PR was actually reverted and the entry should be removed. See also:
```
* Reverted `StaticCustomOp` in favour of adding helper functions `isStatic()`, `getStaticParams()`
  to the `CustomOp` which preserves the same functionality. More specifically, this reverts
  [#1387] and [#1396], modifies [#1484].
```

Reply:
```suggestion
```